### PR TITLE
Add workspace log when terminating hosts

### PR DESCRIPTION
**Pull request by nwalters512**
This may give us better visibility into cases where workspaces launch twice.

### CODE REVIEW

File: cron/workspaceHostTransitions.sql [hunk 1]
```diff
@@ -56,14 +56,28 @@ WITH
...
   )
-UPDATE workspaces AS w
-SET
-  state = 'stopped',
-  stopped_at = NOW(),
-  state_updated_at = NOW()
+INSERT INTO
+  workspace_logs (date, workspace_id, version, state, message)
+SELECT
+  now(),
```
Comment (mwest1066) [inline cron/workspaceHostTransitions.sql:77]: Not that it matters, but all of our `_logs` tables have `DEFAULT now()` for the `date` so we don't have to actually set it.

File: cron/workspaceHostTransitions.sql [hunk 2]
```diff
@@ -56,14 +56,28 @@ WITH
...
+  tw.id,
+  tw.version,
+  tw.state,
+  'Host was terminated'
```
Comment (mwest1066) [inline cron/workspaceHostTransitions.sql:81]:
```suggestion
  'Host was terminated because instance was not found'
```

**Review by mwest1066 — APPROVED** [review]

**Review by mwest1066 — APPROVED** [review]

File: cron/workspaceHostTransitions.sql (hunk 2 above)
Comment (nwalters512) [reply cron/workspaceHostTransitions.sql]: Tweaked in 0ecabcc.

File: cron/workspaceHostTransitions.sql (hunk 1 above)
Comment (nwalters512) [reply cron/workspaceHostTransitions.sql]: Resolved in 0ecabcc.

**Pull request merged**